src/pages/billing/Project.js

```diff
@@ -145,23 +154,38 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
 
   const adminCanEdit = _.filter(({ roles }) => _.includes(billingRoles.owner, roles), projectUsers).length > 1
 
+  const workspacesInProject = useMemo(() => _.flow(
+    _.map(({ workspace }) => workspace),
+    _.filter({ namespace: projectName })
+  )(workspaces), [workspaces, projectName])
+
+  const getBillingAccountStatusIcon = (() => {
+    const allWorkspacesHaveCorrectBillingAccount =
+      _.every({ billingAccount: billingAccountName }, workspacesInProject)
+    return workspace => Utils.cond(
+      [allWorkspacesHaveCorrectBillingAccount, () => null],
+      [billingAccountName === workspace.billingAccount, () => 'success-standard'],
+      ['billingAccountErrorMessage' in workspace, () => 'error-standard'],
```

Comment:
Lodash's `_.has` is more idiomatic for this codebase.

Reply:
Is there not an argument to be made in favour of js primitives vs library calls? Surely `in` is more idiomatic for the language?

Reply:
I generally agree with you, but I would imagine the reason `in` is avoided is because of it's sometimes confusing behavior:
```javascript
> 0 in [1, 2, 3]
true
```
My preference in this particular case is to match the existing style and then have the discussion elsewhere.

Reply:
Thanks for the explanation - I'm new to JS and front end work so the extra context is really helpful.

Reply:
Also, thanks javascript...

Reply:
Wait a second, `_.has` seems to have the same behaviour... 
```javascript
> _.has(0, [1,2,3])
true
```